odoo/addons/base/models/ir_attachment.py

```diff
@@ -697,3 +711,48 @@ def action_get(self):
     def _get_serve_attachment(self, url, extra_domain=None, order=None):
         domain = [('type', '=', 'binary'), ('url', '=', url)] + (extra_domain or [])
         return self.search(domain, order=order, limit=1)
+
+
+class LinkAttachmentMixin(models.AbstractModel):
+    _name = 'link.attachment.mixin'
```

Comment:
The name is confusing: why "link" ?  Simply follow conventions...
```suggestion
    _name = 'ir.attachment.mixin'
```

Reply:
`ir.attachment.mixin` doesn't mean anything to me :confused: 
Maybe `ir.attachment.link`? My thinking is that this mixin adds a link between `ir.attachment` and the model inheriting the mixin